doc/developer-guides/hld/ivshmem-hld.rst

```diff
@@ -98,89 +97,7 @@ MMIO Registers Definition
-add this line as an ``acrn-dm`` boot parameter::
-
-  -s slot,ivshmem,shm_name,shm_size
-
-where
-
--  ``-s slot`` - Specify the virtual PCI slot number
-
--  ``ivshmem`` - Virtual PCI device name
-
--  ``shm_name`` - Specify a shared memory name. Post-launched VMs with the
-   same ``shm_name`` share a shared memory region.
-
--  ``shm_size`` - Specify a shared memory size. The two communicating
-   VMs must define the same size.
-
-.. note:: This device can be used with Real-Time VM (RTVM) as well.
-
-Inter-VM Communication Example
-******************************
-
-The following example uses inter-vm communication between two Linux-based
-post-launched VMs (VM1 and VM2).
-
-.. note:: An ``ivshmem`` Windows driver exists and can be found `here <https://github.com/virtio-win/kvm-guest-drivers-windows/tree/master/ivshmem>`_
-
-1. Add a new virtual PCI device for both VMs: the device type is
-   ``ivshmem``, shared memory name is ``test``, and shared memory size is
-   4096 bytes. Both VMs must have the same shared memory name and size:
-
-   - VM1 Launch Script Sample
-
-     .. code-block:: none
-        :emphasize-lines: 7
-
-        acrn-dm -A -m $mem_size -s 0:0,hostbridge \
-         -s 2,pci-gvt -G "$2" \
-         -s 5,virtio-console,@stdio:stdio_port \
-         -s 6,virtio-hyper_dmabuf \
-         -s 3,virtio-blk,/home/acrn/uos1.img \
-         -s 4,virtio-net,tap0 \
-         -s 6,ivshmem,test,4096 \
-         -s 7,virtio-rnd \
-         --ovmf /usr/share/acrn/bios/OVMF.fd \
-         $vm_name
-
-
-   - VM2 Launch Script Sample
-
-     .. code-block:: none
-        :emphasize-lines: 5
-
-        acrn-dm -A -m $mem_size -s 0:0,hostbridge \
-         -s 2,pci-gvt -G "$2" \
-         -s 3,virtio-blk,/home/acrn/uos2.img \
-         -s 4,virtio-net,tap0 \
-         -s 5,ivshmem,test,4096 \
-         --ovmf /usr/share/acrn/bios/OVMF.fd \
-         $vm_name
-
-2. Boot two VMs and use ``lspci | grep "shared memory"`` to verify that the virtual device is ready for each VM.
-
-   -  For VM1, it shows ``00:06.0 RAM memory: Red Hat, Inc. Inter-VM shared memory (rev 01)``
-   -  For VM2, it shows ``00:05.0 RAM memory: Red Hat, Inc. Inter-VM shared memory (rev 01)``
-
-3. Use these commands to probe the device::
-
-     $ sudo modprobe uio
-     $ sudo modprobe uio_pci_generic
-     $ sudo echo "1af4 1110" > /sys/bus/pci/drivers/uio_pci_generic/new_id
-
-4. Finally, a user application can get the shared memory base address from
-   the ``ivshmem`` device BAR resource
-   (``/sys/class/uio/uioX/device/resource2``) and the shared memory size from
-   the ``ivshmem`` device config resource
-   (``/sys/class/uio/uioX/device/config``).
-
-   The ``X`` in ``uioX`` above, is a number that can be retrieved using the
-   ``ls`` command:
-
-   - For VM1 use ``ls -lh /sys/bus/pci/devices/0000:00:06.0/uio``
-   - For VM2 use ``ls -lh /sys/bus/pci/devices/0000:00:05.0/uio``
+Please refer to :ref:`Enable Inter-VM Communication Based on Ivshmem <enable_ivshmem>`
```

Comment:
Change this to:
```
For usage information, see :ref:`enable_ivshmem`.
```
(That link will be replaced by the title of the document you link to.)